usecase/usecase-webapi-xwalk-tests/samples/IapGooglePlay/iap/iap.html

```diff
@@ -0,0 +1,158 @@
+<html>
+    <head>
+        <meta charset='utf-8'>
+        <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no'/>
+        <style>
+            input.iapButton {
+                border:1px solid black;
+                height:30px;
+                width:40%;
+                margin: 5px;
+            }
+            input.iapDiv {
+                text-align:center;
+            }
+        </style>
+        <title>Sample of In-app purchase</title>
+    </head>
+
+    <script type='text/javascript'>
+        var noticeApple =  "<div style='color:red'>DO NOT USE REAL APPLE ID TO TEST!!!</div>" +
+                           "<div>Use the following account to run this demo:</div>" +
+                           "<div>User name: tom.wang@gmail.com</div>" +
+                           "<div>Password: Intel@123</div>";
```

Comment:
For Apple ID, it is a virtual account, no privacy information, it's OK to be open.
For Google Play account, we will remove it and use personal account.